cmd/infra/aws/proxyca.go

```diff
@@ -0,0 +1,44 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	"golang.org/x/crypto/ssh"
+)
+
+func fetchProxyCA(publicAddr, privateAddr string, privateKey []byte) (string, error) {
+	signer, err := ssh.ParsePrivateKey(privateKey)
+	if err != nil {
+		log.Fatalf("unable to parse private key: %v", err)
+	}
+
+	config := &ssh.ClientConfig{
```

Comment:
same comment about separating the naming and creation logic of CRs in util function